ssl/d1_lib.c

```diff
@@ -20,7 +20,7 @@ static int dtls1_handshake_write(SSL_CONNECTION *s);
 static size_t dtls1_link_min_mtu(void);
 
 /* XDTLS:  figure out the right values */
-static const size_t g_probable_mtu[] = { 1500, 512, 256 };
+static const size_t g_probable_mtu[] = { 1500, 512, 272 };
```

Comment:
I woud like to have a comment here saying it is workaround to make `dtlstest.c:test_swap_records()` happy.